### PR TITLE
PD-1469 / 25.04 / Pd 1469 update static route and aliases content

### PR DESCRIPTION
**Pull request by micjohnson777**
This PR moves static IP address content to the /network/interfaces/_index.md file and deletes the SettingUpStaticIPAddresses.md file. It also deletes the AliasOrStaticIP.md snippet as it is not needed now.

It reorganizes the content in this _index.md file for better readability, and updates screenshots and instructions for the current processes of adding IPs and testing changes.

Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

File: content/SCALE/SCALETutorials/Network/Interfaces/_index.md [hunk 1]
```diff
@@ -14,105 +15,203 @@ tags:
...
 
-## Deleting an Interface
 Click the <i class="material-icons" aria-hidden="true" title="delete">delete</i> icon for the interface.
 A delete interface confirmation dialog opens.
 
-{{< hint type=warning >}}
-Do not delete the primary network interface!
+## Adding Static or Alias IP Addresses
+TrueNAS allows adding static IP addresses to an interface and using them as aliases.
+Static IP addresses set a fixed address for an interface that external devices or websites need to access or remember, such as for VPN access.
```
Comment (DjP-iX) [inline content/SCALE/SCALETutorials/Network/Interfaces/_index.md:128]: I'm not sure if the next few sentences of discription are relevant to the way TrueNAS uses "alias" in the UI (at least not how it uses it when assigning an alias/static IP for the primary interface). I don't know that we need to be too detailed in explaining here, just that this is the procedure for setting a fixed IP address (like the first 1.5 sentences here) on your interface.

File: content/SCALE/SCALETutorials/Network/ConfigureStaticRoutes.md [hunk 2]
```diff
@@ -7,22 +7,26 @@ tags:
 - staticroutes
 ---
 
-TrueNAS does not have defined static routes by default but TrueNAS administrators can use the **Static Routes** widget on the **Network** screen to manually enter routes so a router can send packets to a destination network.
+TrueNAS does not pre-defined static routes by default but TrueNAS administrators can manually add static routes if they want or need to enter routes to a router to send packets to a destination network.
```
Comment (DjP-iX) [inline content/SCALE/SCALETutorials/Network/ConfigureStaticRoutes.md:10]:
```suggestion
TrueNAS does not pre-define static routes by default, but TrueNAS administrators can manually add static routes if they want or need to enter routes to a router to send packets to a destination network.
```

**Review by DjP-iX — CHANGES_REQUESTED** [review]
Just two quick suggestions

File: content/SCALE/SCALETutorials/Network/Interfaces/_index.md (hunk 1 above)
Comment (micjohnson777) [reply content/SCALE/SCALETutorials/Network/Interfaces/_index.md:128]: the request for this change was to be more clear/detailed in explaining this. I didn't think our existing content was far off the mark, but apparently, there was "confusion" about aliases versus static IP addresses.

**Review by DjP-iX — APPROVED** [review]

**Comment by micjohnson777** [comment]
Conflicts resolved.

**Review by DjP-iX — APPROVED** [review]

**Pull request merged**